dm/checker/checker.go

```diff
@@ -119,75 +122,51 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 	rollbackHolder.Add(fr.FuncRollback{Name: "close-DBs", Fn: c.closeDBs})
 
 	c.tctx = tcontext.NewContext(ctx, log.With(zap.String("unit", "task check")))
+
+	// prepare source target do dbs concurrently
+	eg, ctx2 := errgroup.WithContext(ctx)
+	// sourceID => source targetDB mapping, only writing need used mu to avoid concurrent write map
+	var mu sync.Mutex
+	sourceTargetM := make(map[string]map[string][]*filter.Table)
+	for idx := range c.instances {
```

Comment:
May the number of instances be too big to effect performance?

Reply:
Not too slow when external IOs are in parallel